### ISSUE TITLE
Is your ane is able to share multiple files?

### ISSUE THREAD

**Issue by Tpman** [DigitalStrawberry/ANE-Share#5]
Hello!
Thanks for sharing!

Try to use your ane.
 I would like to ask if your ane is able to share multiple files?

like below...
var sharedItems:Vector.<SharedData> = new <SharedData>[];
var len:int = 5;
for (var i:int = 0; i < len; i++) {
 var sharedFile:SharedData; = new SharedData(someFile.nativePath);
sharedFile.isLocalFileUrl = true;
 sharedItems.push(sharedFile);
}
Share.instance.share(sharedItems)

**Comment by marpies** [comment]
Yes, the ANE allows for sharing multiple files, but it is up to the application you select whether it will accept multiple files or not.

We have updated file sharing for Android 7+ in the [v1.0.6 release](https://github.com/DigitalStrawberry/ANE-Share/releases/tag/v1.0.6) so please try that if you experience problems with the older versions.

**Comment by Tpman** [comment]
Hi.

Thank you for your quick answer.

When running version 1.0.5, one image share works well, but multi images were not attached to the files when they were shared.

![image](https://user-images.githubusercontent.com/13846098/54572978-8103bb80-4a2d-11e9-95c0-6b45485616f4.png)

However, there is no response when calling share in version 1.0.6.

![image](https://user-images.githubusercontent.com/13846098/54572983-86f99c80-4a2d-11e9-8be0-824a22c30650.png)

Your help is appreciated.

**Comment by marpies** [comment]
Unfortunately we are unable to reproduce any of the issues you described. Could you share your sample project for us to investigate?
